common/Unit.cpp

```diff
@@ -388,7 +388,11 @@ int UnitBase::uninit()
         dlclose(DlHandle);
     DlHandle = nullptr;
 
+#ifndef NDEBUG
```

Comment:
surely ENABLE_DEBUG ? wow - this is broken:

Makefile.am-if !ENABLE_DEBUG
Makefile.am:AM_CPPFLAGS += -DNDEBUG
Makefile.am-endif

Why do we add a weird synonym of another conditional with a different name; NDEBUG looks un-necessary and grim.